branch_test/copy_phrase_simulation.py

```diff
@@ -0,0 +1,186 @@
+""" This script simulates a copy phrase task. """
```

Comment:
It seems like this is not the right package name for these files. Maybe @sci-tab has a suggestion here.

Reply:
I agree... These would not be installed with the main package as it stands. How about a `bcipy.tasks.simulate` module? 

Then,

- add `synthesized_user.py` to `bcipy.tasks.simulate` as user.py.

I'm flexible on the naming of the module, but that location seems apporatiaite. What do yall think? 

As for this file... we have a `bcipy.tasks.rsvp`. We could add it there under a simulation or tests directory.

Reply:
Actually, this runs like a script, so maybe we can just throw it in a demo? The drawback is we don't catch regressions as easily and it won't be as reusable. Your call, but I'd recommend breaking this into methods.

Reply:
`synthesized_user.py` is renamed as `user.py` and added into `bcipy.tasks.simulate` module.

`copy_phrase_simulation.py` is added into `bcipy.tasks.rsvp.simulation` module.

After solving a few parameter load errors, they will be ready to be merged.